Libraries/oneMKL/sparse_conjugate_gradient/sparse_cg.cpp

```diff
@@ -138,7 +138,7 @@ void run_sparse_cg_example(const sycl::device &dev)
     try {
         mkl::sparse::init_matrix_handle(&handle);
 
-        mkl::sparse::set_csr_data(handle, nrows, nrows, mkl::index_base::zero,
+        mkl::sparse::set_csr_data(main_queue, handle, nrows, nrows, mkl::index_base::zero,
                                           ia_buffer, ja_buffer, a_buffer);
```

Comment:
these are good, but we shold also make one more change to remove deprecated TRSV usage as well -- 

Line 227 - 235 should be changed to the following (added 1.0 before handle for alpha parameter)

```
            {
                mkl::sparse::trsv(main_queue, mkl::uplo::lower,
                                          mkl::transpose::nontrans,
                                          mkl::diag::nonunit, 1.0, handle, r_buffer, t_buffer);
                diagonal_mv<fp, intType>(main_queue, nrows, d_buffer, t_buffer);
                mkl::sparse::trsv(main_queue, mkl::uplo::upper,
                                          mkl::transpose::nontrans,
                                          mkl::diag::nonunit, 1.0, handle, t_buffer, w_buffer);
            }
```

Reply:
Thanks. Fixed